judge/views/organization.py

```diff
@@ -131,8 +132,11 @@ class OrganizationUsers(QueryStringSortMixin, DiggPaginatorMixin, BaseOrganizati
     context_object_name = 'users'
 
     def get_queryset(self):
-        return self.object.members.filter(is_unlisted=False).order_by(self.order).select_related('user') \
-            .defer('about', 'user_script', 'notes')
+        try:
+            return self.object.members.filter(is_unlisted=False).order_by(self.order).select_related('user') \
+                .defer('about', 'user_script', 'notes')
+        except Http404:
+            raise
```

Comment:
What is this doing?

Reply:
This is literally a no-op, please delete @kiritofeng.